package.json

```diff
@@ -25,5 +25,8 @@
     "stylelint-scss": "^3.21.0",
     "webpack": "^5.88.2",
     "webpack-cli": "^5.1.4"
```

Comment:
Kindly install `webpack-dev-server` dependency so as to start webpack server that will serve as the localhost.